tsconfig.typedoc.json

```diff
@@ -24,9 +24,6 @@
     {
       "path": "./packages/javascript-kernel"
     },
-    {
```

Comment:
this seems like a big loss, especially if we're considering making this a more prominent part of the dependency tree.

Reply:
well for now it's just about extracting the JS kernel out of the main bundle.

Let's track that separately since this will likely take more time and thoughts to know what we are considering.